pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -296,7 +296,7 @@ func (m *managerImpl) synchronize(diskInfoProvider DiskInfoProvider, podFunc Act
 	// node conditions report true if it has been observed within the transition period window
 	nodeConditions = nodeConditionsObservedSince(nodeConditionsLastObservedAt, m.config.PressureTransitionPeriod, now)
 	if len(nodeConditions) > 0 {
-		klog.V(3).Infof("eviction manager: node conditions - transition period not met: %v", nodeConditions)
+		klog.V(3).InfoS("Eviction manager: node conditions - transition period not met", "nodeCondition", nodeConditions)
```

Comment:
how the `nodeConditions`  would be formatted? Will `%v` be applied by default?

Reply:
I modified it according to the following rules
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md